### DISCUSSION TITLE
Had to reinstall with 1.1.6, everythning is synced, farming works, but wallet buggy?

### DISCUSSION THREAD

**Discussion by SackofFluor** [Chia-Network/chia-blockchain#5775] [Support]
Hey Guys (and Gals)

Yesterday ui had to reinstall with 1.1.6, everythning is synced, farming works, but my wallet doesnt show any remainxc XCH and my farmer does not show the last block farmed  (and that i farmed a block in general).

Anybody having an idea what i could do, besides reinstalling(and resycning) everything again?

Best
Sack

**Comment by room101-dev** [comment]
The GUI is broken, the Wallet stuff is broken. I quit running the GUI a week ago. I run 1.3 for plotting, and use 1.5 cmd-line on a separate server far away from my plotter. I run cmd line as "chia start -r all", this runs everything, and you can run the cmd line for status.

Do not run "plotman" it is malware. I ran "plotman  interactive" and it did horrible things to my servers, it tried to start plots on every server on my lan, and it killed my plotters, because it tried to start plotter on every single thread of my 32 core cpus. I already knew my max is 4 plots per NVME, so I did't need to have 16 plotting threads running in parallel on each machine. I had already properly configured the .yaml file, but they complete over-wrote it.

So now going forward I will  run no 3rd party sw of any kind from anybody in this 'chia world' of shit.

I just told you re-start your server and let it  be, if you want to know how money you have then just run the cmd-line and dump wallet balances. 

If you must see your balances, then use the chia-blockchain-explorer for 3rd party sites online, and then be certain that its the real balance by consensus, I personally use HPOOL to view my balances on my own personal addresses, but I don't check often, because I dont' care, I just accumulate. To me this is just like mining ETH, where I accumulate, I don't even belong to a single exchange on this planet.

There is no point of using the GUI, especially for plotting the GUI is shit. It's best to have your own bash files, and stagger them and run them to your liking
Signing off, up to you if you want to struggle with bug crap.

I personally even in ETH or BTC never use GUI's I only use cmd-line; Now I will never ever use the chia GUI again, because you have no idea what its really doing.

PLOTMAN said "We're passive, we just real logs, and system stat's", BS they're malware. IMHO CHIA-GUI is also malware.

Since version 1.3 of CHIA-MAIN things got worse instead of better, like I said above I now only use 1.3 for plotting, and I have quit updates for the main-net. I am no longer a beta tester for shit software bugs.

p.s. I did find the chia-pos ( it was listed under HTML ) [creat_plots_disk()C++], and able to mod the c++ file so now my plotter will 'throttle down'  if my NVME's overheat, so I'm no longer killing my NVME's, but still I will not run more than 4x plots per NVME at a time. They can't handle additional abuse. Before my mod my NVME controller was rising up to 90C, and that BS from plotman where they had jacked the threads to 16 actually killed one of my NVME's before I caught it.

Going forward I will  just maintain the source that I need to use, and I will use the min that I have to use.

You understand right that under the cover's here, everything is actually the cmd-line the GUI is just a fools interface to the cmd-line, so unless your a fool, don't use their GUI, I'm almost certain the problems your seeing in the wallet, are GUI induced.

Chia-Dev seems to be using a 3rd party in India contract to implement their install scripts & GUI, there is no other explanation for  the low quality.

**Comment by BrandtH22** [comment]
@SackofFluor , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
